src/main/java/org/folio/service/export/LocalFileSystemExportService.java

```diff
@@ -90,7 +90,7 @@ public void exportInventoryRecords(List<String> inventoryRecords, FileDefinition
   public void postExport(FileDefinition fileDefinition, String tenantId) {
     if (!isValidFileDefinition(fileDefinition)) {
       if (fileDefinition != null && fileDefinition.getJobExecutionId() != null) {
-        errorLogService.saveGeneralError("Export file definition is not valid with id: " + fileDefinition.getId(), fileDefinition.getJobExecutionId(), tenantId);
+        errorLogService.saveGeneralError("Invalid export file definition id: " + fileDefinition.getId(), fileDefinition.getJobExecutionId(), tenantId);
```

Comment:
Why do we have this error message different from the error message below, both of them are checking for the same thing, the difference is just that there is a jobExecutionId, so make both messages consistent

Reply:
sure, will update